server/app.py

```diff
@@ -56,21 +56,14 @@ def __init__(self, track: MediaStreamTrack, pipeline: Pipeline):
         super().__init__()
         self.track = track
         self.pipeline = pipeline
-
-        self._lock = asyncio.Lock()
-        self._fps_interval_frame_count = 0
-        self._last_fps_calculation_time = None
-        self._fps_loop_start_time = time.monotonic()
-        self._fps = 0.0
-        self._fps_measurements = deque(maxlen=60)
-        self._running_event = asyncio.Event()
+        self.fps_meter = FPSMeter()
```

Comment:
@eliteprox this looks like a big change but what I essentually did was just move all the fps stats logic in its own class. If you have any questions don't hesitate to let me know.

Reply:
Nice refactoring, LGTM